### PR TITLE
add ut for pkg/util/net/ip_test.go

### PR DESCRIPTION
**Pull request by hexiaodai**
**Please provide a description of this PR:**

add ut for pkg/util/net/ip_test.go

I want to contribute a little code.

### CODE REVIEW

File: pkg/util/net/ip_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if ipv4, ipv6 := IPsSplitV4V6(tt.ips); !reflect.DeepEqual(ipv4, tt.ipv4) || !reflect.DeepEqual(ipv6, tt.ipv6) {
+				t.Errorf("IPsSplitV4V6() got %v %v, want %v %v", ipv4, ipv6, tt.ipv4, tt.ipv6)
+			}
+		})
+	}
+}
+
+func TestParseIPsSplitToV4V6(t *testing.T) {
```
Comment (wulianglongrd) [inline pkg/util/net/ip_test.go:46]: May be better written as multiple test cases

Comment (hexiaodai) [reply pkg/util/net/ip_test.go:83]: Thanks, I have added multiple test cases

File: pkg/util/net/ip_test.go [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+			ipv6: func() []netip.Addr {
+				ip, _ := netip.ParseAddr("::1")
+				return []netip.Addr{ip}
+			},
+		},
+		{
+			name: "two ipv4",
+			ips:  []string{"1.0.0.0", "255.255.255.255"},
+			ipv4: func() []netip.Addr {
+				ip, _ := netip.ParseAddr("1.0.0.0")
```
Comment (howardjohn) [inline pkg/util/net/ip_test.go:119]: nit: can use MustParseAddr

**Review by howardjohn — APPROVED** [review]

**Pull request merged**